pkgs/top-level/python-packages.nix

```diff
@@ -30195,4 +30233,162 @@ in modules // {
     };
   };
 
+  homeassistant = buildPythonApplication rec {
```

Comment:
An application that is build using Python shouldn't be here. This file is only for Python libraries.